src/protocols/strategies/convex.strategy.ts

```diff
@@ -97,20 +74,15 @@ interface FactoryAPYResonse {
 }
 
 export class ConvexStrategy {
-  static async getValueSources(
-    chain: Chain,
-    vaultDefinition: VaultDefinition,
-    includeBaseEmission: boolean,
-  ): Promise<CachedValueSource[]> {
+  static async getValueSources(chain: Chain, vaultDefinition: VaultDefinition): Promise<CachedValueSource[]> {
     switch (vaultDefinition.vaultToken) {
+      case TOKENS.BCRV_CVXBVECVX:
+        return getLiquiditySources(chain, vaultDefinition);
```

Comment:
we can still define custom overrides where needed, but might need to see how this behaves once the pr is merged...